hello.py

```
@@ -1,5 +1,6 @@
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
+from config import *
```

Ruff (0.11.9)

`from config import *` used; unable to detect undefined names

(F403)